CIP5/README.md

```diff
@@ -48,6 +48,9 @@ We define the following set of common prefixes with their corresponding semantic
 | `vrf_vk`     | VRF verification key                          |
 | `vrf_vkh`    | VRF verification key hash                     |
 | `vrf_sk`     | VRF signing key                               |
+| `script`     | Script hash                                   |
+| `script_vkh` | Verification key hash of used in script       |
+| `script_sk`  | Signing key for vkh used in script            |
```

Comment:
Likewise, we should also include `script_vk` don't you think?

Reply:
added in https://github.com/cardano-foundation/CIPs/pull/31/commits/6843b7b3e1ac401ba90f4bbec776465f45513aff

Reply:
It would be useful if this table had a "Contents" column like so...

```
| Prefix       | Meaning                                         | Contents                  |
| ---          | ---                                             | ---                       |
| `script_vkh` | Verification key hash used in script addresses  | Hash of key               |
| `script_vk`  | Verification key for scripts                    | Verification/public key   |
| `script_sk`  | Signing key for script                          | Signing/private key       |
```

Reply:
done in https://github.com/cardano-foundation/CIPs/pull/31/commits/ea1544309cdefe669a04b403808e878c0f5170af